fact_extractor/plugins/unpacking/generic_carver/test/test_plugin_generic_carver_binwalk.py

```diff
@@ -40,3 +40,16 @@ def test_remove_false_positives(filename):
         shutil.copyfile(TEST_DATA_DIR / filename, test_file)
         ArchivesFilter(temp_dir).remove_false_positive_archives()
         assert test_file.is_file() is False
+
+
+def test_remove_trailing_data():
```

Comment:
It only really works for ".zip" and ".zlib" files that binwalk come out of binwalk. I was unable to create a zlib compresses file that 7z can decompress, though. Probably because zlib is not meant for compressing files by itself. Even the ".zlib" files that come from binwalk are detected as "Type = zip" in 7z so this may be a false positive.

Reply:
huh, that is interesting. So I guess it's good that, at least, we have a bz2 test file now.